lib/linux_admin/ip_addr.rb

```diff
@@ -0,0 +1,31 @@
+module LinuxAdmin
+  class IpAddr
+    include Common
+
+    def ip4
```

Comment:
Should the method be named `ipv4` since that's how people would expect it to be named?